scripts/prepdocs.py

```diff
@@ -314,6 +314,29 @@ def refresh_openai_token():
         openai.api_key = token_cred.get_token("https://cognitiveservices.azure.com/.default").token
         open_ai_token_cache[CACHE_KEY_CREATED_TIME] = time.time()
 
+# recursively read the directory to allow for a subdirectory structure
+def read_files(location, use_vectors):
+    for filename in glob.glob(location):
+        if args.verbose: print(f"Processing '{filename}'")
+        if args.remove:
+            remove_blobs(filename)
+            remove_from_index(filename)
+        elif args.removeall:
```

Comment:
This was already an issue, but should this if here be outside the function?

Reply:
Agree, thought the wrong way here, at least the removeall needs to be outside the function, which already is, so its redundant actually and/or never executed anyways, will be removed!

Reply:
This got fixed!